docs/development/localhost-ssl/README.md

```diff
@@ -84,36 +84,57 @@ The following contains an exemplary configuration:
 
 ```
 
+Save the configuration file, activate it (e.g. with `sudo a2ensite openproject-dev-ssl.conf`) and reload the Apache2
+server. Ensure the syntax is correct with `apachectl configtest`.
 
-
-Save the configuration file, activate it (e.g. with `sudo a2ensite openproject-dev-ssl.conf`) and reload the Apache2 server. Ensure the syntax is correct with `apachectl configtest`. 
-
-In case you're in an environment with SELinux enabled, you will also need to allow Apache2 to connect locally to the application server. You can do that with `/usr/sbin/setsebool -P httpd_can_network_connect 1`.
-
-
+In case you're in an environment with SELinux enabled, you will also need to allow Apache2 to connect locally to the
+application server. You can do that with `/usr/sbin/setsebool -P httpd_can_network_connect 1`.
 
 ## Step 4: Configure OpenProject for HTTPS usage
 
-We assume you have already configured your OpenProject local development environment as [described in this guide](../development-environment-ubuntu). You will need to add your custom host name to `config/environments/development.rb`:
+We assume you have already configured your OpenProject local development environment
+as [described in this guide](../development-environment-ubuntu). You will need to add your custom host name
+to `config/environments/development.rb`:
 
 ```ruby
 Rails.application.config.hosts << 'openproject.example.com'
```

Comment:
This can now be achieved by setting the env var you introduced and should be documented accordingly.
This way one one has to edit their `development.rb` manually.

Reply:
> Extra points for the docker setup.
> 
> Can we perhaps even provide such a `docker-compose.override.yml` that also defines traeffik and stuff?
> 
> Out of scope for this PR perhaps. But I do have one small remark still. Once that's addressed this is good to merge! +1

In order to make it convenient to use for most developers, we want to develop a solution for managing the self signed certificates first. Right now they are auto generated by Traefik and you have to allow them for the backend and the frontend every time they are regenerated.

We already have a plan how to do this. This change here is just to gain a little bit more flexibility to enable this kind of dev setup in the first place.

Reply:
amended docs